Packs/TaniumThreatResponse/Integrations/TaniumThreatResponse/TaniumThreatResponse.yml

```diff
@@ -42,21 +36,30 @@ configuration:
   - suppressed
   required: false
   type: 16
-description: Use the Tanium Threat Response integration to manage endpoints processes, evidence, alerts, files, snapshots, and connections.
+- display: Fetch incidents
+  name: isFetch
+  required: false
+  type: 8
+- defaultvalue: '1'
+  display: Incidents Fetch Interval
+  name: incidentFetchInterval
+  required: false
+  type: 19
```

Comment:
Why did you add this? It is generated automatically and the integration already has it.